src/Discord.Net.Webhook/DiscordWebhookClient.cs

```diff
@@ -91,6 +91,15 @@ private static API.DiscordRestApiClient CreateApiClient(DiscordRestConfig config
             string username = null, string avatarUrl = null, RequestOptions options = null, AllowedMentions allowedMentions = null)
             => WebhookClientHelper.SendMessageAsync(this, text, isTTS, embeds, username, avatarUrl, allowedMentions, options);
 
+        /// <summary> Edits a message posted using this webhook. </summary>
```

Comment:
Missing parameter documentation, follow the same pattern as https://github.com/discord-net/Discord.Net/blob/e3850e1e8f0f24652f22cc6322329495707ff67f/src/Discord.Net.Core/Entities/Messages/IUserMessage.cs#L38-L49